cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -154,8 +157,11 @@ def error_response(message, status):
             content_range = {'start': 0, 'stop': 1, 'end': 2}
 
         # stream out the uploaded files in chunks to disk
-        if int(content_range['start']) == 0:
+        is_initial_import_request = int(content_range['start']) == 0
+        if is_initial_import_request:
             mode = "wb+"
+            set_custom_attribute('course_import_init', True)
```

Comment:
Tracking when the first import request is hit.  `content_range['start']` will be zero for the first chunk only. 

- [ ] Track total course imports attempted.